src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyCommentBeforeLeftCurly.java

```diff
@@ -55,22 +55,22 @@ private class Node {
         }
 
         public Node(int value, Node next)
-        /*********** comment ****/ { // ok until #11410
+        /*********** comment ****/ { // violation ''{' at column 36 should be on the previous line'
             this.value = value;
             this.next = next;
         }
     }
 
     public void array() {
         Integer[] array = null;
-        /*50*/ for/*51*/ (/*52*/Integer/*53*/ i/*54*/:/*55*/ array/*56*/)/*57*/ {/*58*/ // ok
+        /*50*/ for/*51*/ (/*52*/Integer/*53*/ i/*54*/:/*55*/ array/*56*/)/*57*/ {/*58*/
```

Comment:
why you removed "// ok" ? lets keep it.
All inputs are great in their weirdness.

Reply:
sorry i will update the pr while figuring out the issue i have removed for few reason but it is not giving a problem in any case know i also look if any other change of this type has been done

Reply:
done